<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uiSRPowerSupplyDetails</class>
 <widget class="QMainWindow" name="uiSRPowerSupplyDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SR PS Details</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="titleGroupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>750</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
	background-color: rgb(255, 255, 255);
    border: 1px solid black;
    border-radius: 5px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="btnClose">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>20</y>
       <width>85</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>451</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
    border: 1px solid black;
    border-radius: 5px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
    </property>
    <property name="title">
     <string>Main State Machine</string>
    </property>
    <widget class="QSimpleShape" name="ledON">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>24</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="ledType">
      <enum>QSimpleShape::Status</enum>
     </property>
    </widget>
    <widget class="QSimpleShape" name="ledOpen">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>24</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="ledType">
      <enum>QSimpleShape::Status</enum>
     </property>
    </widget>
    <widget class="QSimpleShape" name="ledOFF">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>24</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="ledType">
      <enum>QSimpleShape::Status</enum>
     </property>
    </widget>
    <widget class="QSimpleShape" name="ledStandby">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>24</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="ledType">
      <enum>QSimpleShape::Status</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>50</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Mode</string>
     </property>
    </widget>
    <widget class="QEComboBox" name="cbMode">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>100</width>
       <height>21</height>
      </rect>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):setMode</string>
     </property>
    </widget>
    <widget class="QELabel" name="lblMode">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):getMode</string>
     </property>
    </widget>
    <widget class="QSimpleShape" name="ledFault">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>60</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="colour1" stdset="0">
      <color>
       <red>255</red>
       <green>0</green>
       <blue>0</blue>
      </color>
     </property>
    </widget>
    <widget class="QELabel" name="lblState">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>60</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):getStateHandler</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>60</y>
       <width>70</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Interlock</string>
     </property>
    </widget>
    <widget class="QEPushButton" name="btnOpenloop">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Openloop</string>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):setCommand</string>
     </property>
     <property name="clickText">
      <string>5</string>
     </property>
    </widget>
    <widget class="QEPushButton" name="btnOFF">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>50</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>OFF</string>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):setCommand</string>
     </property>
     <property name="clickText">
      <string>0</string>
     </property>
    </widget>
    <widget class="QEPushButton" name="btnStandby">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Standby</string>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):setCommand</string>
     </property>
    </widget>
    <widget class="QEPushButton" name="btnON">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>20</y>
       <width>40</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>ON</string>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):setCommand</string>
     </property>
     <property name="clickText">
      <string>2</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>291</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
    border: 1px solid black;
    border-radius: 5px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
    </property>
    <property name="title">
     <string>Reference</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>120</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Reference Value</string>
     </property>
    </widget>
    <widget class="QELabel" name="lblReference">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>70</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):getReference</string>
     </property>
    </widget>
    <widget class="QELabel" name="lblCurrent">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):getIload</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>50</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Current</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>154</x>
       <y>50</y>
       <width>55</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Voltage</string>
     </property>
    </widget>
    <widget class="QELabel" name="lblVoltage">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>70</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):getVmon</string>
     </property>
    </widget>
    <widget class="QENumericEdit" name="txtReference">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="displayAlarmStateOption" stdset="0">
      <enum>QE::Never</enum>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):setReference</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="precision">
      <number>3</number>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="addUnits">
      <bool>false</bool>
     </property>
     <property name="writeOnChange">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>451</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
    border: 1px solid black;
    border-radius: 5px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
    </property>
    <property name="title">
     <string>Waveform State Machine</string>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>54</y>
       <width>60</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Trigger</string>
     </property>
    </widget>
    <widget class="QSimpleShape" name="ledSemiAuto">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>24</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="ledType">
      <enum>QSimpleShape::Status</enum>
     </property>
    </widget>
    <widget class="QSimpleShape" name="ledAuto">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>54</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="ledType">
      <enum>QSimpleShape::Status</enum>
     </property>
    </widget>
    <widget class="QSimpleShape" name="ledManual">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>84</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="ledType">
      <enum>QSimpleShape::Status</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>70</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Ramping?</string>
     </property>
    </widget>
    <widget class="QSimpleShape" name="ledRamping">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="ledType">
      <enum>QSimpleShape::Status</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>120</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Next Wavefrom</string>
     </property>
    </widget>
    <widget class="QELabel" name="lblNextWaveform">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>20</y>
       <width>30</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):getWaveformNext</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>120</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Waveform Start</string>
     </property>
    </widget>
    <widget class="QELabel" name="lblStartWaveform">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>50</y>
       <width>30</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):getWaveformStart</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>80</y>
       <width>120</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Waveform Cycle</string>
     </property>
    </widget>
    <widget class="QELabel" name="lblCycleWaveform">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>80</y>
       <width>30</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):getWaveformCycle</string>
     </property>
     <property name="text">
      <string>N/A</string>
     </property>
    </widget>
    <widget class="QELineEdit" name="txtStartWavefrom">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>50</y>
       <width>30</width>
       <height>21</height>
      </rect>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):setWaveformStart</string>
     </property>
    </widget>
    <widget class="QELineEdit" name="txtCycleWaveform">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>80</y>
       <width>30</width>
       <height>21</height>
      </rect>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):setWaveformCycle</string>
     </property>
    </widget>
    <widget class="QEPushButton" name="btnTrigger">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>110</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Trigger</string>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):setCommand</string>
     </property>
     <property name="clickText">
      <string>12</string>
     </property>
    </widget>
    <widget class="QEPushButton" name="btnResume">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>140</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Resume</string>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):setCommand</string>
     </property>
     <property name="clickText">
      <string>11</string>
     </property>
    </widget>
    <widget class="QEPushButton" name="btnPause">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>110</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Pause</string>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):setCommand</string>
     </property>
     <property name="clickText">
      <string>10</string>
     </property>
    </widget>
    <widget class="QEPushButton" name="btnStop">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>140</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):setCommand</string>
     </property>
     <property name="clickText">
      <string>9</string>
     </property>
    </widget>
    <widget class="QEPushButton" name="btnAuto">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Auto</string>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):setCommand</string>
     </property>
     <property name="clickText">
      <string>7</string>
     </property>
    </widget>
    <widget class="QEPushButton" name="btnManual">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Manual</string>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):setCommand</string>
     </property>
     <property name="clickText">
      <string>8</string>
     </property>
    </widget>
    <widget class="QEPushButton" name="btnSemiAuto">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Semiauto</string>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):setCommand</string>
     </property>
     <property name="clickText">
      <string>6</string>
     </property>
    </widget>
    <widget class="QEPushButton" name="qepushbutton">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>20</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Skip</string>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):setCommand</string>
     </property>
     <property name="clickText">
      <string>15</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>110</y>
      <width>291</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
    border: 1px solid black;
    border-radius: 5px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
    </property>
    <property name="title">
     <string>Reference</string>
    </property>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>70</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Tuning</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>80</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Faults</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>80</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Readings</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnFirmwareSettings">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>170</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Firmware Settings</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnWaveforms">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>160</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Waveform Settings</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>60</y>
       <width>80</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Plotter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>140</y>
       <width>120</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Postmortem</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QEComboBox</class>
   <extends>QComboBox</extends>
   <header>QEComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>QELabel</class>
   <extends>QLabel</extends>
   <header>QELabel.h</header>
  </customwidget>
  <customwidget>
   <class>QELineEdit</class>
   <extends>QLineEdit</extends>
   <header>QELineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>QENumericEdit</class>
   <extends>QWidget</extends>
   <header>QENumericEdit.h</header>
  </customwidget>
  <customwidget>
   <class>QEPushButton</class>
   <extends>QPushButton</extends>
   <header>QEPushButton.h</header>
  </customwidget>
  <customwidget>
   <class>QSimpleShape</class>
   <extends>QWidget</extends>
   <header>QSimpleShape.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
